Cargo.toml

```diff
@@ -10,9 +10,8 @@ edition = "2018"
 crate-type = ["cdylib"]
 
 [dependencies]
-k8s-openapi = { version = "0.14.0", features = ["v1_20"] }
-kubewarden-policy-sdk = "0.4.1"
+k8s-openapi = { version = "0.15.0", features = ["v1_24"] }
```

Comment:
Unfortunately not, because of two reasons:

1. Not all the Rust policies might need the k8s-openapi crate. For example, a policy doing "jq-like" queries would not need it. TBH, we could solve that by using some feature flag, but...
2. The policy SDK is a Rust library that uses `k8s-openapi`. The crate prohibits libraries consuming `k8s-openapi` to specify which Kubernetes version has to be used. This is done to grant more freedom to the end users of the library. This forces us to make the decision inside of the policy itself. We cannot workaround that, the `k8s-openapi` is checking for this misconfiguration and raises an error at build time

Reply:
ah I see, thanks for the explanation